client/src/game/Game.js

```diff
@@ -1,6 +1,11 @@
 import React, { useEffect, useRef } from "react";
-import { gameLoop, player, DIRECTIONS } from "./GameLoop";
-import movePlayer from "./movePlayer";
+import {
+  gameLoop,
+  player,
+  keyState,
+  DIRECTIONKEYS,
```

Comment:
Usually, you separate uppercase constants with an underline:
https://javascript.info/variables#uppercase-constants

```suggestion
  DIRECTION_KEYS,
```